src/components/settings/DataManagement/index.tsx

```diff
@@ -1,47 +1,75 @@
-import InfoIcon from '@/public/images/notifications/info.svg'
-
 import Track from '@/components/common/Track'
 import { SETTINGS_EVENTS } from '@/services/analytics'
-import { Paper, Grid, Typography, Tooltip, SvgIcon, Button } from '@mui/material'
+import { Paper, Grid, Typography, Button } from '@mui/material'
 import { useState } from 'react'
 import ImportAllDialog from '../ImportAllDialog'
+import { getPersistedState } from '@/store'
+import { SAFE_EXPORT_VERSION } from '../ImportAllDialog/useGlobalImportFileParser'
+import { pendingTxsSlice } from '@/store/pendingTxsSlice'
+import { cookiesSlice } from '@/store/cookiesSlice'
+
+const handleExport = () => {
+  const filename = `safe-data-${new Date().toISOString().slice(0, 10)}.json`
+
+  // Exclude pending transactions and cookie consent from export
+  const { [pendingTxsSlice.name]: _p, [cookiesSlice.name]: _c, ...exportData } = getPersistedState()
```

Comment:
As we now know what exact data we want to import, should we change the export to also just include that needed data? 
 *  address book
 *  added Safes
 *  safeApps
 *  settings
 *  session